### PR TITLE
Enable cargo-spellcheck for blockchain/message_pool 

### LINKED ISSUE #1689: Enable `cargo-spellcheck` for `blockchain/message_pool`
**Issue summary**

Make sure that `cargo spellcheck` can run without errors in `blockchain/message_pool`. For details, see the tracking issue.

**Other information and links**

### PR DESCRIPTION
**Pull request by tyshko5**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes #1689

**Other information and links**

### CODE REVIEW

File: .config/forest.dic [hunk 1]
```diff
@@ -17,3 +17,7 @@ tuple
 mutex
 RPC
 parameterize/D
+сonfig
```
Comment (lemmih) [inline .config/forest.dic:20]: `config` is not a word. Try perhaps `configuration`.

File: .config/forest.dic [hunk 2]
```diff
@@ -17,3 +17,7 @@ tuple
...
+TODO
```
Comment (lemmih) [inline .config/forest.dic:21]: `TODO` should never be part of the official Forest documentation. Move it to a dev comment or refactor it some other way.

File: .config/forest.dic [hunk 3]
```diff
@@ -17,3 +17,7 @@ tuple
...
+struct
```
Comment (lemmih) [inline .config/forest.dic:22]: `struct` is not a word. Maybe use `structure`.

File: .config/forest.dic [hunk 4]
```diff
@@ -17,3 +17,7 @@ tuple
...
+Config
```
Comment (lemmih) [inline .config/forest.dic:23]: `Config` is not a word. Try perhaps `Configuration`.

**Review by lemmih — CHANGES_REQUESTED** [review]
Good start. Try not to add new words to the dictionary unless absolutely necessary.

**Review by lemmih — CHANGES_REQUESTED** [review]
Remove the addition to the dictionary and it's good to go.

A lot of the documentation is in serious need of a re-write but that's beyond the scope of this PR.

**Comment by lemmih** [comment]
Gosh darn it, cla-bot. Why you always causing trouble?

**Review by lemmih — APPROVED** [review]

File: blockchain/message_pool/src/config.rs [hunk 5]
```diff
@@ -1,6 +1,8 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+#[allow(unused_imports)]
+use crate::MessagePool;
```
Comment (LesnyRumcajs) [inline blockchain/message_pool/src/config.rs:5]: why do we need an unused import?

File: blockchain/message_pool/src/msgpool/msg_pool.rs [hunk 6]
```diff
@@ -391,9 +391,9 @@ where
         Ok(publish)
     }
 
-    /// Finish verifying signed message before adding it to the pending mset hashmap. If an entry
-    /// in the hashmap does not yet exist, create a new mset that will correspond to the from
-    /// message and push it to the pending hashmap.
+    /// Finish verifying signed message before adding it to the pending `mset` hash-map. If an entry
```
Comment (LesnyRumcajs) [inline blockchain/message_pool/src/msgpool/msg_pool.rs:394]: `hashmap` should be fine, no? https://en.wiktionary.org/wiki/hashmap

File: blockchain/message_pool/src/config.rs (hunk 5 above)
Comment (lemmih) [reply blockchain/message_pool/src/config.rs:5]: Good catch. I totally missed that. We shouldn't import `MessagePool` here. Instead, the documentation should refer to `crate::MessagePool`.

File: blockchain/message_pool/src/msgpool/msg_pool.rs (hunk 6 above)
Comment (lemmih) [reply blockchain/message_pool/src/msgpool/msg_pool.rs:394]: We can add it to the custom dictionary.

File: blockchain/message_pool/src/config.rs (hunk 5 above)
Comment (tyshko5) [reply blockchain/message_pool/src/config.rs]: Fixed this

**Review by LesnyRumcajs — APPROVED** [review]

**Pull request merged**